### PR TITLE
vendored gardener v1.83.3

### PR DESCRIPTION
**Pull request by kon-angelo**
**How to categorize this PR?**

/area control-plane
/kind enhancement
/platform openstack

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other dependency
Vendor gardener/gardener =>  `v1.83.3`
```

### CODE REVIEW

**Review by shafeeqes — APPROVED** [review]
Thanks!
/lgtm

File: go.mod [hunk 1]
```diff
@@ -7,9 +7,9 @@ require (
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/ahmetb/gen-crd-api-reference-docs v0.3.0
 	github.com/coreos/go-systemd/v22 v22.5.0
-	github.com/gardener/etcd-druid v0.19.2
-	github.com/gardener/gardener v1.81.6
-	github.com/gardener/machine-controller-manager v0.48.1
+	github.com/gardener/etcd-druid v0.20.1
+	github.com/gardener/gardener v1.83.2
```
Comment (ialidzhikov) [inline go.mod:11]: Is it possible to update to v1.83.3? In v1.83.3 an issue in the extension library that prevents CPM for hibernated Shoots is resolved.

Comment (kon-angelo) [reply go.mod]: adapted the PR 👍

**Review by AndreasBurger — DISMISSED** [review]
lgtm

**Pull request merged**